electronicparsers/openmx/parser.py

```diff
@@ -716,6 +753,41 @@ def parse(self, mainfile: str, archive: EntryArchive, logger):
         sec_run.program = Program(
             name='OpenMX', version=str(mainfile_parser.get('program_version'))
         )
+        # Stress tensor values are not written to outfile but only to stdout.
+        # That means we have to check if the stdout is actually present and
+        # look for the stress values there.
+        stress_tensor_enabled = mainfile_parser.get('scf.stress.tensor')
+        stdout_uele_list = []
+        stress_tensors = None
+        is_stdout_valid = False
+        if stress_tensor_enabled == "on":
+            mainfile_dir = path.dirname(mainfile)
+            stdout_path = None
+            for file in listdir(mainfile_dir):
+                f_path = path.join(mainfile_dir, file)
+                if path.isfile(f_path):
+                    with open(f_path) as input_file:
+                        for index, line in enumerate(input_file):
+                            if r'Welcome to OpenMX' in line:
+                                stdout_path = f_path
+                                break
+                            if index == 500:
+                                break
+                if stdout_path is not None:
+                    break
+
+            if stdout_path is not None:
+                stdout_parser.mainfile = stdout_path
+                stdout_parser.parse()
+                elapsed_time_stdout = stdout_parser.get('elapsed.time')
+
+                if elapsed_time_mainfile == elapsed_time_stdout:
```

Comment:
Add comment please:

It is possible there are multiple calculations in the same directory and thus multiple stdout files as well, we need to find the correct one to look for the stress values, a simple test is to check the total runtime and later also the Uele concsistency.